### ISSUE TITLE
Allow availability to be shared / publicly visible

### ISSUE THREAD

**Issue by Axel-Naumann** [indico/newdle#189]
Hi,

I'd be nice if users could opt in into sharing their availability with everyone with the poll link. That way, other participants can see progress.

It would be nice if the number of participants that have been participating so far could be shown either way, to anyone with the poll URL, to understand progress of the meeting preparation.

Cheers, Axel.

**Comment by ThiefMaster** [comment]
I think this is almost a duplicate of #174.

I don't think it should be up to the participants whether or not everyone can see the progress; I'd rather make this the default, possibly with an option for the organizer to opt-out (like on Doodle) since seeing the others' answers is incredibly useful.

**Comment by Axel-Naumann** [comment]
Thanks, and apologies, I had overlooked #174 and I was submitting two suggestions in one issue :-)

For "let others see what I answered" (i.e. personal info as shown in the owner's status page) I was suggesting a conservative change - I assumed that it's preferred for privacy reasons, which would suggest an opt-in by participants.

I'd be very happy to have an anonymous progress shown to all participants. IIUC only this part is covered by #174 .
